paddle/fluid/operators/detail/brpc_server.cc

```diff
@@ -13,31 +13,34 @@
 // limitations under the License.
 
 #include "paddle/fluid/operators/detail/brpc_server.h"
+#include "paddle/fluid/operators/detail/brpc_sendrecvop_utils.h"
+#include "paddle/fluid/operators/detail/brpc_variable_response.h"
 #include "paddle/fluid/operators/detail/request_handler.h"
 
 namespace sendrecv {
 
-typedef std::unordered_map<std::string,
-                           paddle::operators::detail::RequestHandler*>
-    HandlerMap;
+namespace detail = paddle::operators::detail;
+
+typedef std::unordered_map<std::string, detail::RequestHandler*> HandlerMap;
 
 class BRPCServiceImpl : public SendRecvService {
  public:
   explicit BRPCServiceImpl(const HandlerMap& rpc_call_map)
       : request_send_h_(nullptr),
         request_get_h_(nullptr),
         request_prefetch_h_(nullptr) {
-    auto it = rpc_call_map.find(paddle::operators::detail::kRequestSend);
+    VLOG(3) << "BRPCServiceImpl size: " << rpc_call_map.size();
+    auto it = rpc_call_map.find(detail::kRequestSend);
```

Comment:
In fact, you can also use brpc to do the method mapping.

Reply:
Please give me a example.

Reply:
For each service you define in the "proto" file, you can add more than one method. Please refer to https://github.com/Tuvie/brpc/blob/master/example/http_c%2B%2B/http.proto